src/validation.cpp

```diff
@@ -2923,7 +2925,7 @@ static bool FindUndoPos(CValidationState &state, int nFile, FlatFilePos &pos, un
     bool out_of_space;
     size_t bytes_allocated = UndoFileSeq().Allocate(pos, nAddSize, out_of_space);
     if (out_of_space) {
-        return AbortNode(state, "Disk space is low!", _("Error: Disk space is low!"));
+        return AbortNode(state, "Disk space is too low!", _("Error: Disk space is too low!"), CClientUIInterface::MSG_NOPREFIX);
```

Comment:
Since you are changing the translation here anyway, might as well remove the prefix and not set the flag?

Reply:
Please see:
- https://github.com/bitcoin/bitcoin/pull/15894#pullrequestreview-230889663 by @ryanofsky 
- https://github.com/bitcoin/bitcoin/pull/15894#issuecomment-487466226 by @promag 
- https://github.com/bitcoin/bitcoin/pull/15894#issuecomment-487468327 by @jonasschnelli

Reply:
Yeah, I'd say it is shorter to write 

```cpp
return AbortNode(state, "Disk space is too low!", _("Disk space is too low!"));
```

which should do the same?

Reply:
> ... which should do the same?

Not quite.
The main idea is not to translate {"Error: "|"Warning: ":} prefixes in the `debug.log` and `stderr`.

Reply:
> which should do the same?

Not quite.
The difference is whether {"Error: "|"Warning: ":} prefix is translated or not in the `debug.log` and `stderr`.

Reply:
Right. I guess it shouldn't, but at the current state of this pull request, it is translated.

I presume you are fixing this in #16224 (or similar)?

Reply:
Yes. I think it was my fault to touch translations in this PR ((